ras_party/controllers/respondent_controller.py

```diff
@@ -216,6 +221,31 @@ def change_respondent_details(respondent_data, respondent_id, session):
         change_respondent(respondent_data)
 
 
+@with_db_session
+def get_respondents_by_survey_and_business_id(survey_id: UUID, business_id: UUID, session: session) -> list:
+    """
+    Gets a list of Respondents enrolled in a survey for a specified business
```

Comment:
This did stump me for a moment. But it's more a consequence of the business concept of an enrolment, and the fact that an enrolled respondent can separately have a status of `ENABLED` or `DISABLED`. Although we don't test for it explicitly, this implementation correctly returns respondents with _all_ enrolment statuses (as required by the calling service) 👍